features/schedule/basic_workflow/feature.go

```diff
@@ -0,0 +1,122 @@
+package basic_workflow
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+	"go.temporal.io/api/enums/v1"
+	"go.temporal.io/api/workflowservice/v1"
+	"go.temporal.io/sdk-features/harness/go/harness"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/workflow"
+)
+
+var Feature = harness.Feature{
+	Workflows: BasicScheduleWorkflow,
+	Execute:   Execute,
+}
+
+func BasicScheduleWorkflow(ctx workflow.Context, arg string) (string, error) {
+	return arg, nil
+}
+
+func Execute(ctx context.Context, r *harness.Runner) (client.WorkflowRun, error) {
+	// Start schedule every 2s
+	workflowID := uuid.NewString()
+	handle, err := r.Client.ScheduleClient().Create(ctx, client.ScheduleOptions{
+		// TODO(cretz): Required? https://github.com/temporalio/sdk-go/issues/957
+		ID:      uuid.NewString(),
+		Spec:    client.ScheduleSpec{Intervals: []client.ScheduleIntervalSpec{{Every: 2 * time.Second}}},
```

Comment:
I see more risks of bugs across SDKs with the structured calendar format than the interval spec format... Can we consider defining this test using an equivalent structured calandar spec format instead? For example (using TS syntax): `second: { start: 0, end: 59, step: 2 }, minute: '*', hour: '*', ...`

Reply:
That sounds like a new test. I can think of dozens and dozens of tests if we need to test each input form. These are the initial basic tests, not meant to validate each field.

Reply:
> I can think of dozens and dozens of tests if we need to test each input form. These are the initial basic tests, not meant to validate each field.

I might have not expressed myself clearly here... I totally agree that testing tons of inputs on schedules in features tests would just be insane and would not bring in much value.

My argument was rather that, from a cross SDK point of view, if we have to retain only a very minimal number of test inputs for schedule testing, then it seems to me that there would be more value in testing some structured calandar input (because it is more likely to expose divergence between SDKs) rather than the rather trivial interval format. The specific calandar I proposed is functionally equivalent to the "every 2 seconds" interval, which means that testing methodology remains the same.

But yes, that can definitely be a distinct test... Your call.

Reply:
Hrmm, I was going for testing how a user might write "every X interval" schedule.